1 - recursion, chaining, es5 classes/ex-1/second-homework.js

```diff
@@ -0,0 +1,114 @@
+// First exercise:
+const games = [
+    {name: 'alibaba', type: 'pacanele', jackpot: false},
+    {name: 'alibaba2', type: 'live', jackpot: false},
+    {name: 'gonzo cautare', type: 'poker', jackpot: false},
+    {name: 'alibaba', type: 'live', jackpot: true},
+    {name: 'dracula', type: 'pacanele', jackpot: false},
+    {name: 'dracula', type: 'pacanele', jackpot: true,
+        children: [
+            {name: 'cosmic fortune', type: 'pacanele', jackpot: true},
+            {name: 'gonzo cautare', type: 'poker', jackpot: false},
+        ]
+    }
+];
+
+let flatten = function flatten(array) {
```

Comment:
naming conflict. If used like this, you don't need to name the function. It should not be declared with let either since this should not be reassigned later.